### PR TITLE
Updated UNIT Tests needed for content title change

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf [hunk 1]
```diff
@@ -1,347 +0,0 @@
-[
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf]: Why is this add-on un-installed? why is this file deleted?

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/version.sf [hunk 2]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/version.sf]: Why is this add-on un-installed? why is this file deleted?

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf [hunk 3]
```diff
@@ -1,6979 +0,0 @@
-{
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf]: Why is this add-on un-installed? why is this file deleted?

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Controllers/BaseJobProfileWidgetController.cs [hunk 4]
```diff
@@ -73,44 +73,6 @@ public string GetDynamicTitle(bool skipNoTitle)
...
-        public string CheckForAcronym(string title)
-        {
-            return IsSpecialConditionWords(title)
-                ? title
-                : title.Split(' ').Aggregate(string.Empty, (current, next) => $"{current} {ChangeWordCase(next)}").Trim();
-        }
-
-        // Further investigation on implmenting Special Title Case for certain JobTitles will be done in the story - 6426
-        // https://skillsfundingagency.atlassian.net/browse/DFC-6426
-        public bool IsSpecialConditionWords(string word)
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Controllers/BaseJobProfileWidgetController.cs]: Only this function should have been replaced with CMS functionality.

**Comment by vallabha24** [comment]
With the CMS functionality event the acronyms are covered under special case. And from what Karl mentioned it looks like the SIT doesn't have any changes made from 6044
